tests/vmi_networking_test.go

```diff
@@ -756,11 +756,14 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 					ipv4NetworkCIDR = api.DefaultVMCIDR
 				}
 				By("Checking ping to gateway")
-				pingVirtualMachine(serverVMI, gatewayIPFromCIDR(ipv4NetworkCIDR), "\\$ ")
+				err = testPingFromVMI(serverVMI, gatewayIPFromCIDR(ipv4NetworkCIDR), "\\$ ")
+				Expect(err).ToNot(HaveOccurred(), "VMI %s was not able to ping gateway address %s", serverVMI.Name, gatewayIPFromCIDR(ipv4NetworkCIDR))
```

Comment:
No need for the message now, it is embedded in the error.
You could also move to "Succeed()".

Reply:
Thanks. I forgot to apply the changes also in vmi_networking_test.
Done.